TestE2E/PAR/BUILD/build_encrypt_ inspecting wrong descriptor

### Version of Singularity:

What version of Singularity are you using? Run:

```
3.6.0-rc.4
```

### Actual behavior

Tests are checking for encrypted rootfs as descriptor 2 - but it will be 3 now

```
--- FAIL: TestE2E/PAR/BUILD/build_encrypt_with_PEM_file (32.39s)
                singularitycmd.go:663: Running command "/usr/local/bin/singularity build --encrypt --pem-path /tmp/stest.618035292/pem-pub-046172184 /tmp/stest.618035292/pem-encryption.405935990/encrypted_cmdline_option.sif library://alpine:latest"
                singularitycmd.go:663: Running command "/usr/local/bin/singularity sif info 2 /tmp/stest.618035292/pem-encryption.405935990/encrypted_cmdline_option.sif"
                singularitycmd.go:663: failed to match pattern: Command "/usr/local/bin/singularity sif info 2 /tmp/stest.618035292/pem-encryption.405935990/encrypted_cmdline_option.sif":
                    Expect output stream contains:
                    Fstype:    Encrypted squashfs
                    Command output stream:
                    Descr slot#: 1
                      Datatype:  JSON.Generic
                      ID:        2
                      Used:      true
                      Groupid:   1
                      Link:      NONE
                      Fileoff:   131072
                      Filelen:   95
                      Ctime:     2020-06-15 11:24:23 -0400 EDT
                      Mtime:     2020-06-15 11:24:23 -0400 EDT
                      UID:       0
                      Gid:       0
                      Name:      oci-config.json
```

--- a/e2e/imgbuild/imgbuild.go
+++ b/e2e/imgbuild/imgbuild.go
@@ -810,7 +810,7 @@ func (c imgBuildTests) buildDefinition(t *testing.T) {
 }
 
 func (c *imgBuildTests) ensureImageIsEncrypted(t *testing.T, imgPath string) {
-	sifID := "2" // Which SIF descriptor slots contains encryption information
+	sifID := "3" // Which SIF descriptor slot contains the (encrypted) rootfs
 	cmdArgs := []string{"info", sifID, imgPath}
 	c.env.RunSingularity(
 		t,

